src/pages/docs/deployments/kubernetes/kustomize/index.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2023-01-01
+modDate: 2023-01-01
+title: kustomize  
+description: Use kustomize to deploy resources to a Kubernetes cluster.
+navOrder: 9
+---
+
+Octopus supports the deployment of Kubernetes resources through the `kustomize` step.
+
+This step allows you to source your kustomize files from git, perform variable substitutions based on your environment and/or tenant and finally apply the changes to your Kubernetes clusters.
+
+[Kustomize](https://kustomize.io) introduces a template-free way to customize application configuration that simplifies the use of off-the-shelf applications.
+
+## Recommended usages for this step
+
+We list a few different scenarios below to help you figure out what is the best setup for you to deploy your applications to a Kubernetes cluster with Octopus.
+
+*When you use Kustomize for one application configuration*
+
+1. **The kustomize way**  
```

Comment:
I would be cautious to create a "kustomize way vs Octopus way" split here. It is useful for internal discussions. But in public docs I think we should use something neutral, such as "Using multiple overlays / using a single overlay for Octopus".

Reply:
sounds good, can you update doc